docs/painless/painless-contexts/painless-runtime-fields-context.asciidoc

```diff
@@ -12,10 +12,22 @@ about how to use runtime fields.
         Accepts the values from the script valuation. Scripts can call the
         `emit` method multiple times to emit multiple values.
 +
---
+.Signatures of `emit`
+[%collapsible%open]
+====
+`emit`'s signature depends on the `type` of the field.
+
+[[emit-boolean]]   `boolean`::   `emit(boolean)`
+[[emit-date]]      `date`::      `emit(long)`
+[[emit-double]]    `double`::    `emit(double)`
+[[emit-geo-point]] `geo_point`:: `emit(double lat, double lon)`
+[[emit-ip]]        `ip`::        `emit(String)`
+[[emit-long]]      `long`::      `emit(long)`
+[[emit-keyword]]   `keyword`::   `emit(String)`
+
 IMPORTANT: This method cannot accept `null` values. Do not call this method if
 the referenced fields do not have any values.
---
+====
```

Comment:
I'm not super happy with how this looks when rendered. I'd prefer a more compact representation but everything that looks like it could be smaller has tons of padding.